integrations/Posthog/browser.js

```diff
@@ -68,6 +70,10 @@ class Posthog {
       property_blacklist: this.propertyBlackList,
       disable_cookie: this.disableCookie,
     };
+
+    if (POSTHOG && POSTHOG.loaded) {
+      configObject.loaded = POSTHOG.loaded;
+    }
```

Comment:
We can add a check if loaded is a function or not, as we are expecting it as a function. Otherwise, it looks good.

Reply:
Sounds good. @shrouti1507 please take care of it.

Reply:
But let's do that only if PostHog fails to initialize if the `loaded` parameter is not a function. I would assume PostHog would already have this validation.

Reply:
Checked sending `page` events to `posthog`. When we set a function inside `loaded` for example `posthog.identify('1234')` , it is identifying the user along with creating a page view. Now, when we set `loaded: "ABC"`, it is creating a single page view ( without throwing any error). That means, posthog is internally discarding the erroneous format of loaded variable. @ItsSudip

Reply:
Ok, then we don't need to validate it as posthog is already discarding from their end.